.github/workflows/python_release.yml

```diff
@@ -0,0 +1,23 @@
+name: Python Package
+on:
+  push:
+    tags: "*"
```

Comment:
will this be on every release or every tag push? lets have it on release

Reply:
done